package.json

```diff
@@ -29,13 +36,21 @@
     "react-redux": "^7.2.1",
     "react-scripts": "^3.4.3",
     "redux": "^4.0.5",
-    "typescript": "^3.9.7"
+    "typescript": "^3.9.7",
+    "concurrently": "^5.3.0",
+    "cross-env": "^7.0.2",
+    "electron-is-dev": "^1.2.0",
+    "wait-on": "^5.2.0"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
+    "electron-build": "electron-builder",
+    "release": "yarn build && electron-builder --publish=always",
+    "e-build": "yarn build && yarn electron-build",
```

Comment:
Czy są jakieś zalety yarna przed npm-em ? 
Bo ja nie mam go zainstalowanego i w tym miejscu u mnie pojawia się wyjątek ( nie mniej jednak, jeżeli będzie to potrzebne, to mogę sobie zainstalować )
![image](https://user-images.githubusercontent.com/21079319/98850821-a1c82280-2455-11eb-875f-1fc702a00a99.png)

Reply:
nie ma różnicy w sume dla nas, już podmieniam